app/v2/models/response/retrieveSelfAssessmentBalanceTransactions/BalanceDetails.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.models.response.retrieveSelfAssessmentBalanceTransactions
```

Comment:
Not sure if you saw the discussion on Slack, but is it possible to use the `retrieveBalanceTransactions` package name instead?  (Planning to rename the existing v2 model packages for consistency, as there aren't very many of them)

Reply:
I've also updated the other (single) package